From 87811f34faa30bad2ef9841cfcbd12c98a129497 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Thu, 30 Jan 2025 16:42:31 +0100
Subject: [PATCH 1484/1695] core: psa-adac: fix compilation issue due to
 static_buffer_msg.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the following errors:
core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h:16:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]
   16 | int psa_adac_static_buffer_msg_release();
      | ^~~
core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h:18:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]
   18 | request_packet_t *psa_adac_static_buffer_msg_get_request();
      | ^~~~~~~~~~~~~~~~
core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h:19:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]
   19 | response_packet_t *psa_adac_static_buffer_msg_get_response();

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3acb755f69fb1816d667cb6c9df4fdd8a40928f7 core: psa-adac: fix compilation issue due to static_buffer_msg.h]
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I918090aba2c0ebc3153f54093c8d9a760c08c869
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/433584
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 .../psa-adac/transport_layer/transports/static_buffer_msg.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h b/core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h
index 2064c46e4..6996eaab6 100644
--- a/core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h
+++ b/core/lib/psa-adac/transport_layer/transports/static_buffer_msg.h
@@ -13,9 +13,9 @@
 #include <stdint.h>
 
 int psa_adac_static_buffer_msg_init(uint8_t *buffer, size_t size);
-int psa_adac_static_buffer_msg_release();
+int psa_adac_static_buffer_msg_release(void);
 
-request_packet_t *psa_adac_static_buffer_msg_get_request();
-response_packet_t *psa_adac_static_buffer_msg_get_response();
+request_packet_t *psa_adac_static_buffer_msg_get_request(void);
+response_packet_t *psa_adac_static_buffer_msg_get_response(void);
 
 #endif //PSA_ADAC_STATIC_BUFFER_MSG_H
-- 
2.47.2

